internal/manifest/docker_schema2.go

```diff
@@ -0,0 +1,13 @@
+package manifest
+
+import (
+	"github.com/opencontainers/go-digest"
+)
+
+// Schema2Descriptor is a “descriptor” in docker/distribution schema 2.
```

Comment:
```suggestion
// Schema2Descriptor is a “descriptor” in docker/distribution schema 2.
//
// This is publicly visible as c/image/manifest.Schema2Descriptor.
```

… and similar comments to the other types and public functions that are referenced from c/image/manifest.

---

All edits to `internal/manifest/docker_schema2.go` should apply to `internal/manifest/oci_index.go` analogously.